README.md

```diff
@@ -29,10 +29,29 @@ You can use `set_veth.sh` to set veths for testing the application.
 After the script is executed, veth0(10.0.0.4) and veth1(10.0.0.5) are created.
 veth1 is created in `test_namespace` namespace but veth0 in local.
 
-To execute ARP response example, follow the next steps:
-1. `sudo set_veth.sh`
-2. execute `sudo ./arp_response veth0 2048 1024 64 64 64 64` in `/target/release/examples`
-3. `ip netns exec test_namespace arping -I veth1 10.0.0.4`
+echo example has these options.
+
+  -i, --interface <interface>  Interface to use.  
+  -s, --chunk <size>           Chunk size [default: 2048].  
+  -c, --chunk <count>          Chunk count [default: 1024].  
+  -r, --rx <count>             Rx ring size [default: 64].  
+  -r, --tx <count>             Tx ring size [default: 64].  
+  -r, --fill <count>           Fill ring size [default: 64].  
+  -r, --completion <count>     Completion ring size [default: 64].  
+
+To execute echo example, follow the next steps:
+1. `sudo ./set_veth.sh`
+2. execute `sudo ./echo -i veth0` in `/target/release/examples`
+3. echo example has three functions, arp reply, icmp echo(ping), udp echo. 
+
+if you want to test arp reply execute the following command  
+`sudo ip netns exec test_namespace arping 10.0.0.4`. 
+
+if you want to test icmp echo execute the following command.  
```

Comment:
First letter should always be capital letter, and comma is required for this sentence.
```suggestion
If you want to test icmp echo, Execute the following command.  
```